### PR TITLE
change include_los_displacement to include_displacement_maps, use thi…

### PR DESCRIPTION
**Pull request by cirrusasf**
…s option to control if or not to include both line of sight (los) and vertical displacement maps in the product

### CODE REVIEW

**Comment by asjohnston-asf** [comment]
Looks right. There's one test that is failing that should be updated at https://github.com/ASFHyP3/hyp3-gamma/blob/develop/tests/test_entrypoints.py#L29 , and we'll need to add a CHANGELOG entry.

We should wait to merge this into develop until after the GAMMA upgrade has been released to production. Since this is a breaking change, we'll also need to coordinate merging to develop and main with corresponding changes to hyp3, hyp3-sdk, hyp3-docs, hyp3-testing, and Vertex (at a minimum).

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,26 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
...
 
-## [4.9.1](https://github.com/ASFHyP3/hyp3-gamma/compare/v4.9.0...v4.9.1)
+## [5.0.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v4.9.0...v5.0.0)
 
-### Changed
-* InSAR readme and xml templates have been updated to reflect the change in displacement map options and to include more information on adjusting unwrapped phase and displacement map values relative to a user-selected reference point.
-* Updated the URL for the [Copernicus DEM](https://spacedata.copernicus.eu/explore-more/news-archive/-/asset_publisher/Ye8egYeRPLEs/blog/id/434960) information webpage.
+### Added
+* An `--include-displacement-maps` argument to the `insar` entrypoint which will cause both the
+  line-of-site and vertical displacement maps to be included.
```
Comment (asjohnston-asf) [inline CHANGELOG.md:14]:
```suggestion
  line-of-sight and vertical displacement maps to be included.
```

**Review by asjohnston-asf — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,26 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
...
+* InSAR README and XML templates have been updated to reflect the change in displacement map options
+  and to include more information on adjusting unwrapped phase and displacement map values relative
+  to a user-selected reference point.
+* Updated the URL for the [Copernicus DEM](https://spacedata.copernicus.eu/explore-more/news-archive/-/asset_publisher/Ye8egYeRPLEs/blog/id/434960)
+  information webpage.
+
+### Deprecated
+* The `--include-los-displacment` argument to the `insar` entrypoint is depreciated and may be removed
+  at any time. `--include-displacement-maps` should be used instead as setting either option to `true`
+  will cause both the line-of-site and vertical displacement maps to be included.
```
Comment (jhkennedy) [inline CHANGELOG.md:29]:
```suggestion
  will cause both the line-of-sight and vertical displacement maps to be included.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,26 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
...
+  has changed to `include_displacement_maps`. Now, both the line-of-sight displacement and the vertical
+  displacement maps will be included if this option is set to `True`.
+* InSAR README and XML templates have been updated to reflect the change in displacement map options
+  and to include more information on adjusting unwrapped phase and displacement map values relative
+  to a user-selected reference point.
+* Updated the URL for the [Copernicus DEM](https://spacedata.copernicus.eu/explore-more/news-archive/-/asset_publisher/Ye8egYeRPLEs/blog/id/434960)
+  information webpage.
+
+### Deprecated
+* The `--include-los-displacment` argument to the `insar` entrypoint is depreciated and may be removed
```
Comment (hjkristenson) [inline CHANGELOG.md:27]: Should depreciated be changed to deprecated to match the header?

Comment (jhkennedy) [reply CHANGELOG.md]: yes, yes it should.

**Pull request merged**